supermarq/benchmarks/bacon_shor_code.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import cirq
+
+
+def get_code_cycle(rows: int, cols: int) -> cirq.Circuit:
```

Comment:
These benchmarks look great! It'll be nice to have a few more ECC tailored apps.

First thing, define a `BaconShorCode` class to represent the benchmark. See for example the [GHZ benchmark](https://github.com/SupertechLabs/SupermarQ/blob/main/supermarq/benchmarks/ghz.py).

Reply:
The `__init__()` function should take in all of the appropriate benchmark parameters as inputs.